src/wire/ip.rs

```diff
@@ -71,6 +73,8 @@ impl fmt::Display for Protocol {
             Protocol::Udp => write!(f, "UDP"),
             Protocol::Ipv6Route => write!(f, "IPv6-Route"),
             Protocol::Ipv6Frag => write!(f, "IPv6-Frag"),
+            Protocol::Esp => write!(f, "ESP"),
+            Protocol::Ah => write!(f, "AH"),
```

Comment:
Could you change these to `IPsec-ESP` and `IPsec-AH`? I think that's clearer.

Reply:
I changed the display output. Should the enum variant also be changed?

Reply:
Yes, maybe it should indeed!

Reply:
It's done